domains/authorization/Authorization.API/API/Authorization/[Features]/GrantConsentToClientCommand.cs

```diff
@@ -65,7 +65,7 @@ public async Task Handle(GrantConsentCommand command, CancellationToken cancella
     }
 }
 
-public record GrantConsentCommand(Guid UserId, string OrganizationCvr, IdpClientId IdpClientId) : IRequest;
+public record GrantConsentToClientCommand(Guid UserId, string OrganizationCvr, IdpClientId IdpClientId) : IRequest;
 
-public class UserNotAffiliatedWithOrganizationCommandException()
-    : ForbiddenException("Not authorized to perform action");
+public class UserNotAffiliatedWithOrganizationCommandException() : ForbiddenException("Not authorized to perform action");
```

Comment:
Not a big fan of in file exception types. Do we need this exception type? Would a throw Forbidden exception with unique description be enough?

Reply:
Done, using ForbiddenException directly.